python/rikai/spark/functions/vision.py

```diff
@@ -133,3 +142,43 @@ def video_to_images(
         )
         for idx, img in enumerate(video_iterator)
     ]
+
+
+@udf(returnType=ImageType())
+def spectrogram_image(video, size: int = 224) -> Image:
+    import ffmpeg
```

Comment:
this import should be written below docstring.

Reply:
sounds good